src/ConfigOptions.php

```diff
@@ -0,0 +1,14 @@
+<?php
```

Comment:
Ich würde gern alles im .json halten. Die Config ist eigentlich eine gute Möglichkeit, value objects einzusetzen.

Reply:
Wir hatten ja gesprochen, dass wir die Konfiguration von VOG separieren von der Definition der Value Objects.
Das letztere ist ja in einer oder mehrerer JSON Dateien. Ersteres kann so global für mehrere json Dateien vorgenommen werden. Ich habe das so gelöst damit dann va. keine String Vergleiche für die Feature Toggles im Code gemacht werden müssen. siehe ValueObjectBuilder.php Zeile 330 ff z.B.

Reply:
Ich würde gern einen mischmasch aus config-Formaten vermeiden. Solche array Konstruktionen wie auf Z. 330 können mit value objects ja gut vermieden werden. Ich würde die Config-Datei im json lassen und in ein value object parsen, dass als Properties enums mit den Konfigurationsparametern hat.

Reply:
Ist umgestellt. wobei ich noch keine gute lösung gefunden habe das json in ein vo zu parsen.

Reply:
Alles, ich würde das jetzt erstmal mergen und dann den Rest dann nachziehen. Gibt ja keinen Autorelease aus dem Master